lib/datastore/query.js

```diff
@@ -75,7 +75,16 @@ function Query(namespace, kinds) {
 }
 
 /**
+ * @param {boolean=} autoPaginateVal - Have pagination handled automatically.
+ *     Default: true.
  * @return {module:datastore/query}
+ *
+ * @example
+ * // Have pagination handled automatically
+ * var autoPaginated = query.autoPaginate();
```

Comment:
Can we expand this to show what the code would look like when using these? Feels like this is one of those docs for `getFoo` that says "gets foo property".